.docksal/src_files/Gruntfile.js

```diff
@@ -0,0 +1,18 @@
+module.exports = function(grunt) {
+  // Load all plugins and tasks defined by the grunt-drupal-tasks package.
+  require('grunt-drupal-tasks')(grunt);
+  grunt.loadNpmTasks('grunt-deploy-site');
+  grunt.loadNpmTasks('grunt-symlink');
+  grunt.config.set('deploy_site.development.options.branch', 'SOE-SOEOP-368');
```

Comment:
The default here probably should be `soe-develop-mc`. Future improvement would be to take this as an argument or from a variable.